### DISCUSSION TITLE
Finish 2024-06 and Start 2024-09

### DISCUSSION THREAD

**Discussion by merks** [eclipse-simrel/simrel.build#428] [Announcements]
The 2024-06 release is final:

- https://download.eclipse.org/releases/2024-06
- https://download.eclipse.org/releases/latest

Download are available:

- https://www.eclipse.org/downloads/packages

---

The 2024-09 release is started:

- https://download.eclipse.org/releases/2024-09
- https://download.eclipse.org/releases/milestone

The 2024-09 staging repository exists:

- https://download.eclipse.org/staging/2024-09

The 2024-09 schedule is as follows:

https://github.com/eclipse-simrel/.github/blob/main/wiki/SimRel/2024-09.md

---

Orbit M0 milestones repositories for 2024-09 / 4.33 have been created.

Consult the following table for a summary overview of the updated available versions:

- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/table.html

Aggregation - All of below

- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/2024-09
- https://download.eclipse.org/tools/orbit/simrel/orbit-aggregation/milestone/latest

BND - Wrapped

- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/2024-09
- https://download.eclipse.org/tools/orbit/simrel/maven-bnd/nightly/latest

OSGi - Direct from Maven

- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/2024-09
- https://download.eclipse.org/tools/orbit/simrel/maven-osgi/milestone/latest

Native Signed

- https://download.eclipse.org/tools/orbit/simrel/maven-sign/2024-09
- https://download.eclipse.org/tools/orbit/simrel/maven-sign/milestone/latest

Legacy - Build from Source

- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/2024-09
- https://download.eclipse.org/tools/orbit/simrel/orbit-legacy/milestone/latest

---

Jetty's latest release is 12.0.10:

https://download.eclipse.org/tools/orbit/simrel/maven-jetty/release/latest

**Comment by pcdavid** [comment]
Dates for 2024-09: https://github.com/eclipse-simrel/.github/blob/main/wiki/SimRel/2024-09.md

**Reply by merks** [reply]
Good idea.  I've added it above so that I'll include it also in future announcements.
